Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -148,17 +148,16 @@ public protocol ManifestLoaderDelegate {
 /// serialized form of the manifest (as implemented by `PackageDescription`'s
 /// `atexit()` handler) which is then deserialized and loaded.
 public final class ManifestLoader: ManifestLoaderProtocol {
```

Comment:
👀 this is the key change in the PR, basically adding an in-memory cache for manifests, making things more thread-safe and easier to reason about

Reply:
That makes sense.  This looks like a good cleanup even without the concurrency.